docs/api-docs/partner/create-a-sandbox.md

```diff
@@ -1,23 +1,45 @@
-# Create a Sandbox Store
+# Create a Sandbox or Trial Store
 
- 
+A sandbox store is intended for developing and testing apps without the 15-day time limit of a trial store. It cannot be process any transactions.
 
-A sandbox store cannot process transactions and is for developing and testing apps without the 15-day time limit of a trial store. 
+This article walks you through the steps of creating a sandbox store and explains how to open a support case if you encounter any issues.
 
-This article will walk you through the steps of creating a sandbox store.
+## Creating a sandbox or trial store
 
-## Creating a sandbox store
+**To create a sandbox or trial store**
 
-1. Go to the [BigCommerce Pricing](https://www.bigcommerce.com/essentials/pricing/) page and select **Start your free trial**.
-2. When prompted for an email address, enter the same email you used when applying to become a partner.
-3. In the trial store setup form, enter a store name, password, and other details. Then select **Create your store**.
-4. Log into your Partner account to access the [Partner Portal](https://partners.bigcommerce.com). 
-5. Navigate to **Support > Contact Support** and select **Create a Support Case**.
-6. Select the type of issue you have (Product Support or Billing Support).
-7. Select **Yes** to the question "Is this related to a specific store?"
-6. While completing the form, use "Partner Sandbox" as the subject line. In the case description, enter your new sandbox store’s canonical URL (for example: http://your-store-name.mybigcommerce.com) and request conversion to a sandbox. 
-7. Under **Issue Type**, select **Merchant Account Services**. Click **Create Case**. We will flag this new trial store as a sandbox to ensure your store access continues beyond the 15-day trial period. This process can take several days.
-8.  Log into your new trial store using the same email associated with your partner login.
+1. Go to the [BigCommerce Partner Portal](https://partners.bigcommerce.com/English/) and sign in using your credentials.
+2. After signing in, click **Create New**, and select an option:
+    * **Deal Registration w/store**: select this option to create a sandbox with a deal registration.
+    * **Partner Use Sandbox**: select this option to create a sandbox for your own internal use.
+    * **Deal Registration**: select this option to create a deal registraion. Note that you cannot request sandboxes using this option.
+3. If you select **Deal Registration w/ Store**, complete all fields in the **Create a Store/Register a Deal** page to create a sandbox.
+    * In the **Create a Store** field, select **Partner Sandbox** or **30 Day Trial**.
+    * After completing all fields, click **Submit**.
+
+After submitting the registration for **Deal Registration w/Store**, you can track your deal's progress in the **Deal Registration & Manage Stores Dashboard**. Allow up to 15 minutes for the new store/trial to appear in these lists.
```

Comment:
I'm unable to verify this on the Partner Dashboard. The only page I could find was the "Review Deal Registrations" page.